src/main/java/uk/ac/cam/cl/dtg/isaac/api/AssignmentFacade.java

```diff
@@ -525,8 +526,10 @@ public Response getAssignmentProgressDownloadCSV(@Context final HttpServletReque
             if (null != formatMode && formatMode.toLowerCase().equals("excel")) {
                 headerBuilder.append("\uFEFF");  // UTF-8 Byte Order Marker
             }
+            SimpleDateFormat dateFormat = new SimpleDateFormat("EEE dd MMM yyyy HH:mm:ss z");
+            dateFormat.setTimeZone(GMT);
             headerBuilder.append(String.format("Assignment (%s) Results: Downloaded on %s \nGenerated by: %s %s \n\n",
```

Comment:
In the next function (`getGroupAssignmentsProgressDownloadCSV`), in line 899, we have some similar code. It may be worth to use the same dateFormat there as well. If you do that, it would make sense to make this dateFormat a private variable and then initialise it in the constructor (or something like that), so that you don't have to repeat the same code twice.